eras/conway/impl/cddl-files/conway.cddl

```diff
@@ -434,7 +434,7 @@ transaction_witness_set =
   , ? 1: nonempty_set<native_script>
   , ? 2: nonempty_set<bootstrap_witness>
   , ? 3: nonempty_set<plutus_v1_script>
-  , ? 4: [+ plutus_data ]
+  , ? 4: nonempty_set<plutus_data>
```

Comment:
The deserialisation still doesn't fail if there are duplicates,  but that's more an implementation detail - this type reflects the set tag,which we need anyway. 
I will update the ticket https://github.com/IntersectMBO/cardano-ledger/issues/3922
to mention uncommenting the test we have for this: https://github.com/IntersectMBO/cardano-ledger/blob/master/eras/alonzo/impl/testlib/Test/Cardano/Ledger/Alonzo/Binary/TxWitsSpec.hs#L114

Reply:
Yes. Thank you.